### PR TITLE
feat: change create app commands

### PR DESCRIPTION
**Pull request by avneesh0612**

### CODE REVIEW

**Review by jnsdls — APPROVED** [review]
LGTM

File: components/contract-tabs/code/ContractCode.tsx [hunk 1]
```diff
@@ -67,9 +67,8 @@ const INSTALL_COMMANDS = {
 };
 
 const CREATE_APP_COMMANDS = {
-  evm: "npx thirdweb@latest create --app",
-  solana:
-    "npx thirdweb@latest create --template next-typescript-solana-starter",
+  evm: "npx thirdweb@latest create app --evm",
```
Comment (nachoiacovino) [inline components/contract-tabs/code/ContractCode.tsx:70]: this should be --app for both
```suggestion
  evm: "npx thirdweb@latest create --app --evm",
```

File: components/contract-tabs/code/ContractCode.tsx [hunk 2]
```diff
@@ -67,9 +67,8 @@ const INSTALL_COMMANDS = {
...
+  solana: "npx thirdweb@latest create app --solana",
```
Comment (nachoiacovino) [inline components/contract-tabs/code/ContractCode.tsx:71]:
```suggestion
  solana: "npx thirdweb@latest create --app --solana",
```

**Review by nachoiacovino — CHANGES_REQUESTED** [review]

**Review by nachoiacovino — APPROVED** [review]

File: components/contract-tabs/code/ContractCode.tsx [hunk 3]
```diff
@@ -67,9 +67,8 @@ const INSTALL_COMMANDS = {
 };
 
 const CREATE_APP_COMMANDS = {
-  evm: "npx thirdweb@latest create --app",
-  solana:
-    "npx thirdweb@latest create --template next-typescript-solana-starter",
+  evm: "npx thirdweb@latest create --app --evm",
```
Comment (nachoiacovino) [inline components/contract-tabs/code/ContractCode.tsx:70]:
```suggestion
  evm: "npx thirdweb@latest create app --evm",
```

File: components/contract-tabs/code/ContractCode.tsx [hunk 4]
```diff
@@ -67,9 +67,8 @@ const INSTALL_COMMANDS = {
...
+  solana: "npx thirdweb@latest create --app --solana",
```
Comment (nachoiacovino) [inline components/contract-tabs/code/ContractCode.tsx:71]:
```suggestion
  solana: "npx thirdweb@latest create app --solana",
```

**Pull request merged**